src/gobexport/exporter/config/meetbouten.py

```diff
@@ -86,13 +88,51 @@ class RollagenExportConfig:
         $$AK25$$|1|121287|485235|POINT (121287.0 485245.0)
 
     """
+    row_count = 0
+
+    def row_formatter_rollagen(row):
+        node = row['node']
+
+        # Convert the bouwblok geometry to a centroid
+        geom = shape(node['isGemetenVanBouwblok']['edges'][0]['node']['geometrie'])
+        node['geometrie'] = mapping(geom.centroid)
+
+        # Round the coordinates to three decimal places
+        rounded_coordinates = []
+        for coordinate in node['geometrie']['coordinates']:
+            rounded_coordinates.append(str(round(coordinate, 3)))
+        node['geometrie']['coordinates'] = rounded_coordinates
```

Comment:
`node['geometrie']['coordinates'] = [str(round(coordinate, 3)) for coordinate in node['geometrie']['coordinates']]`

Reply:
Will do